tests/unit/accordion/accordion_core.js

```diff
@@ -7,15 +7,23 @@ module( "accordion: core", setupTeardown() );
 
 $.each( { div: "#list1", ul: "#navigation", dl: "#accordion-dl" }, function( type, selector ) {
 	test( "markup structure: " + type, function() {
```

Comment:
Markup structure refers to the structural classes and their relation to each other that are part of our API. These are fixed and not controlled by the user.